docs/react/react-native.md

```diff
@@ -76,3 +76,64 @@ export function useRefreshOnFocus<T>(refetch: () => Promise<T>) {
 ```
 
 In the above code, `refetch` is skipped the first time because `useFocusEffect` calls our callback on mount in addition to screen focus.
+
+## Disable re-renders on out of focus Screens
+
+In some situations, including performance concerns, you may want to stop re-renders when a React Native screen gets out of focus. To achieve this we can use `useFocusEffect` from `@react-navigation/native` together with the `notifyOnChangeProps` query option.
```

Comment:
I have a question if you may, when the screen gets out of focus, does effects clean up get called ?

Reply:
Hi @incepter!

Yes, [useFocusEffect](https://reactnavigation.org/docs/use-focus-effect/) runs the cleanup whenever a screen/component gets out of focus :+1: